src/_reducers/TickReducer.js

```diff
@@ -18,22 +28,24 @@ export default (state = initialState, action) => {
                 epoch: +tick.epoch,
                 quote: +tick.quote,
             };
-            return state.update(symbol, List.of(), v => v.takeLast(60).push(newTick));
+            return state.update(symbol, List.of(), v => v.takeLast(4999).push(newTick));
```

Comment:
this is the storing part, if we do not store all of them, the dynamic data loading would not work properly, 4999 certainly is an arbritary number where we could adjust, but it will still be large.

let's use this number and test it out ? better than wild guessing what is the best 

Or even better if you have alternative approach?